external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaDontFailOnDataLossSuite.scala

```diff
@@ -221,7 +221,7 @@ class KafkaSourceStressForDontFailOnDataLossSuite extends StreamTest with KafkaM
       .as[(String, String)]
     val query = startStream(kafka.map(kv => kv._2.toInt))
 
-    val testTime = 1.minutes
+    val testTime = 20.seconds
```

Comment:
@dilipbiswal . nit. Although this is a random stress test, can we compare the numbers of Kafka operation of before and after in general? Although the test time is reduced to 1/3, line 255 seems to recover it.

Reply:
@dongjoon-hyun Actually i did count the number of times we went inside each case block. With this change, we did a little less than we used to do before.. but if i remember, we did enough kafka operations. I will get the count of each case block tomorrow for your perusal.

Reply:
Thank you for confirmation. Then, it's okay.